ts/Core/Axis/AxisDefaults.ts

```diff
@@ -3157,47 +3149,6 @@ namespace AxisDefaults {
      * @apioption zAxis
      */
 
```

Comment:
This is now followed by a closing curly bracket and so is removed by TypeScript before JSDoc and DTS generator can pick it up. You can move it outside to the root level followed by `''; // keeps doclets above in JS file` / or to the top inside the scope.